src/test/java/edu/kis/powp/jobs2d/TestJobs2dApp.java

```diff
@@ -47,6 +47,9 @@ private static void setupCommandTests(Application application) {
 
 		application.addTest("Run command", new SelectRunCurrentCommandOptionListener(DriverFeature.getDriverManager()));
 
+		application.addTest("Visitor Command Counting test", new DriverCommandCounterVisitorTest(DriverFeature.getDriverManager().getCurrentDriver()));
+
+		application.addTest("Visitor Command Executor test", new DriverCommandExecutorVisitorTest(DriverFeature.getDriverManager().getCurrentDriver()));
```

Comment:
The driver passed to the test may be obsolete (you should always use the current driver)